--- a/src/Makefile
+++ b/src/Makefile
@@ -27,7 +27,7 @@
 			--host=$(CROSS_HOST) --enable-embedded --disable-shared \
 			--disable-tests ac_cv_header_sys_select_h=no \
 			--disable-tests ac_cv_header_sys_socket_h=no \
-		CFLAGS="-Os -ffunction-sections -I$(TOPDIR)/target/firmware/include -nostartfiles -nodefaultlibs"
+		CFLAGS="-Os -ffunction-sections -nostartfiles -nodefaultlibs"
 
 shared/libosmocore/build-target/src/.libs/libosmocore.a: shared/libosmocore/build-target/Makefile
 	cd shared/libosmocore/build-target && make

--- a/src/Makefile
+++ b/src/Makefile
@@ -11,28 +11,9 @@
 
 TOPDIR=$(shell pwd)
 
-all: libosmocore-target nofirmware firmware mtk-firmware
+all: nofirmware firmware mtk-firmware
 nofirmware: layer23 osmocon gsmmap
 
-libosmocore-target: shared/libosmocore/build-target/src/.libs/libosmocore.a
-
-shared/libosmocore/configure: shared/libosmocore/configure.ac
-	cd shared/libosmocore && autoreconf -fi
-
-shared/libosmocore/build-target:
-	mkdir $@
-
-shared/libosmocore/build-target/Makefile: shared/libosmocore/configure shared/libosmocore/build-target
-	cd shared/libosmocore/build-target && ../configure \
-			--host=$(CROSS_HOST) --enable-embedded --disable-shared \
-			--disable-tests ac_cv_header_sys_select_h=no \
-			--disable-tests ac_cv_header_sys_socket_h=no \
-		CFLAGS="-Os -ffunction-sections -nostartfiles -nodefaultlibs"
-
-shared/libosmocore/build-target/src/.libs/libosmocore.a: shared/libosmocore/build-target/Makefile
-	cd shared/libosmocore/build-target && make
-
-
 .PHONY: osmocon
 osmocon: host/osmocon/osmocon
 
